framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/database/driver/Db2ExternalDriver.java

```diff
@@ -31,4 +33,9 @@ public String getImageName() {
     public String getImageVersion() {
         return "11.1.4.4";
     }
+
+    @Override
+    public String getJdbcDriverUrl() {
```

Comment:
DB2 can be built without external driver file. We can use it.

Reply:
Thanks for spotting! Will revert this change.